### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jasonblais**
Update README in preparation for sharing with SAs to enable them present existing version in customer demos, and for sharing with community to empower them with help wanted tickets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,113 @@
...
+  - `/jira settings notifications [on/off]` - Set whether Direct Message notifications are sent for assignments and comments in assigned issues.
+  - `/jira view <issue-key>` - View a Jira issue specified by <issue-key>.  
+
+## 3. Installation
+
+### 3.1 Jira Cloud (Core + Software)
+
+#### 3.1.1 Mattermost
+
+1. Download this binary: // XXX
```
Comment (jasonblais) [inline README.md]: Need to add link to binary here

File: README.md [hunk 2]
```diff
@@ -1,19 +1,113 @@
-# mattermost-plugin-jira ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-jira/master.svg) ![Codecov branch](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jira/master.svg)
+# Mattermost Jira Plugin ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-jira/master.svg) ![Codecov branch](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-jira/master.svg)
```
Comment (hanzei) [inline README.md]: Jira or JIRA?

Comment (jasonblais) [reply README.md]: `Jira`. Atlassian rebranded `JIRA` to `Jira`

Comment (levb) [reply README.md]: https://www.atlassian.com/legal/trademark I was under the wrong impression that the trademarked name is "JIRA", but evidently I was wrong and we should use "Jira"

Comment (levb) [reply README.md]: I am totally going to get rid of `JIRA` in the code now, will look so much better

Comment (jasonblais) [reply README.md]: @levb All good. The old plugin code had `JIRA` because that's what they used before.

File: README.md (hunk 1 above)
Comment (levb) [reply README.md]: https://s3.amazonaws.com/mattermost-public-plugins-kubernetes/jira-test-2.0.0.tar.gz should work

**Pull request merged**